tests/test_volcano_plot.py

```diff
@@ -203,12 +208,11 @@ def assert_callback(p_value, nclicks, input_value):
     )
 
 
-def test_effect_size_line_input_value(dash_threaded, selenium):
+def test_effect_size_line_input_value(dash_threaded):
     """Modifies the effect_size line value."""
```

Comment:
This test fails because the function converting the string to an array raises a ValueError